app/src/main/res/values-fr/strings.xml

```diff
@@ -0,0 +1,186 @@
+
+	<!-- Folder select -->
+	<string name="title_activity_folder_select">Selectionner Fichier</string>
+	<string name="navigation_here">Ici</string>
+	<string name="navigation_read_only">Lecture uniquement</string>
+
+	<!-- Saving messages -->
+	<string name="toast_save_successful">Sauvegardé avec succès.</string>
+	<string name="toast_save_in_progress">Sauvegarde en cours…</string>
+	<string name="toast_save_failed">Sauvegarde échouée.</string>
+	<string name="toast_save_init_error">Erreur de chargement des lieus de sauvegarde.</string>
+	<string name="toast_save_init_no_locations">Veuillez ouvrir l\'application pour ajouter des lieus de sauvegarde.</string>
+	<string name="toast_save_file_exists_overwrite">Ecrasement du fichier présent à cet endroit…</string>
+	<string name="toast_save_file_exists_fail">Impossible de sauvegarder, ce fichier existe déjà.</string>
+	<string name="toast_save_file_exists_self_abort">Abandon! Il semblerait que vous remplacez ce fichier par lui-même.</string>
+	<string name="toast_save_file_exists_self_continue">Attention! Il semblerait que vous
+		remplacez ce fichier par lui-même. Comme permi par le comportement de sauvegarde, écrasement du fichier…</string>
+	<string name="downloader_description">Sauvegarde de %1$s</string>
+
+	<!-- Debug / Error -->
+	<string name="not_supported_content_type">Type de contenu inacceptable</string>
+	<string name="not_supported" tools:keep="@string/not_supported">Veuillez &lt;a href="%1$s">faire une demande d\'ajout de fonctionnalité&lt;/a> avec ce
+		type de contenu afin que &author; puisse ajouter un support officiel. &lt;br/>&lt;br/>Vous pouvez également activer le paramètre \"sauvegarde forcée\"
+		setting pour forcer &appname; à essayer de le sauvegarder.</string>
+
+	<!-- Loading -->
+	<string name="loading_title">Test du support…</string>
+	<string name="loading_body">Veuillez patienter tandis que je test le téléchargement du support…</string>
+
+	<!-- Credits -->
+	<string name="credits_version">v%1$s</string>
+	<string name="title_activity_credits">A propos de &appname;</string>
+	<string name="logo_description">Logo</string>
+	<string name="credits_creator">par <a href="https://michael.standen.link">&author;</a></string>
+	<string name="credits_content1">&appname; est une simple application pour sauvegarder
+		des fichiers sur votre téléphone via la fonctionnalité partager.
+	</string>
+	<string name="credits_content2">Vous pouvez accéder au code source, faire connaitre des bugs, et demander des fonctionnalités sur
+		<a href="https://github.com/ScreamingHawk/phone-saver">Github</a>.
+	</string>
+	<string name="credits_content3">Cette application est complètement open source; elle a pour license la
+		<a href="https://tldrlegal.com/license/mit-license">license MIT</a>.
+	</string>
+	<string name="credits_content_translator">Cette traduction de &appname; a été proposée par
+		<a href="https://michael.standen.link">&author;</a>.
```

Comment:
You should absolutely change this to yourself!!! It's there to credit the kind souls like yourself who take the time to contribute to open source :)